core/src/components/datetime/test/sub-pixel-width/index.html

```diff
@@ -42,6 +42,25 @@ <h2>Modal</h2>
       </ion-modal>
     </ion-content>
   </ion-app>
+
+  <script>
+    const openModal = document.getElementById('open-modal');
+
+    const observer = new MutationObserver(mutationRecords => {
```

Comment:
Could we pull this to a util file somewhere so other tests can import it? (And also make the event name more generic?)

Reply:
Updated, let me know if this aligns with what you were thinking 👍

Reply:
Looks great, thanks!